vtex/toolbelt
# Update Typescript version

#### What is the purpose of this pull request?
- Update `typescript` version to `3.7.3`
- Update `eslint`, `prettier` and `eslint-config-vtex`
- Format with `prettier` and fix `eslint` errors

#### How should this be manually tested?

Install this branch:
```
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout refact/use-esModuleInterop && \
yarn && yarn global add file:$PWD
```
Check if it compiles and passes all tests. You can test around using some commands.

#### Types of changes
